src/java.base/share/classes/java/lang/invoke/MethodType.java

```diff
@@ -1165,10 +1165,12 @@ Invokers invokers() {
      * Convenience method for {@link #methodType(java.lang.Class, java.lang.Class[]) methodType}.
      * Any class or interface name embedded in the descriptor string will be
      * resolved by the given loader (or if it is null, on the system class loader).
-     * <p>
-     * Note that it is possible to encounter method types which cannot be
+     *
+     * @apiNote
+     * It is possible to encounter method types which cannot be
      * constructed by this method, because their component types are
-     * not all reachable from a common class loader.
+     * not all reachable from a common class loader, or some component types
```

Comment:
"reachable from a common class loader" pre-dates your change but I think it should say visible to the given class loader.